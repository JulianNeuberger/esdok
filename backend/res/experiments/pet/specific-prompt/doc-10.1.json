{"text": "The MPON sents the dismissal to the MPOO . The MPOO reviews the dismissal . The MPOO opposes the dismissal of MPON or the MPOO confirmes the dismissal of the MPON .", "name": "doc-10.1", "id": "doc-10.1", "category": "", "tokens": [{"text": "The", "indexInDocument": 0, "posTag": "DT", "sentenceIndex": 0}, {"text": "MPON", "indexInDocument": 1, "posTag": "NNP", "sentenceIndex": 0}, {"text": "sents", "indexInDocument": 2, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 3, "posTag": "DT", "sentenceIndex": 0}, {"text": "dismissal", "indexInDocument": 4, "posTag": "NN", "sentenceIndex": 0}, {"text": "to", "indexInDocument": 5, "posTag": "TO", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 6, "posTag": "DT", "sentenceIndex": 0}, {"text": "MPOO", "indexInDocument": 7, "posTag": "NNP", "sentenceIndex": 0}, {"text": ".", "indexInDocument": 8, "posTag": ".", "sentenceIndex": 0}, {"text": "The", "indexInDocument": 9, "posTag": "DT", "sentenceIndex": 1}, {"text": "MPOO", "indexInDocument": 10, "posTag": "NNP", "sentenceIndex": 1}, {"text": "reviews", "indexInDocument": 11, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 12, "posTag": "DT", "sentenceIndex": 1}, {"text": "dismissal", "indexInDocument": 13, "posTag": "NN", "sentenceIndex": 1}, {"text": ".", "indexInDocument": 14, "posTag": ".", "sentenceIndex": 1}, {"text": "The", "indexInDocument": 15, "posTag": "DT", "sentenceIndex": 2}, {"text": "MPOO", "indexInDocument": 16, "posTag": "NNP", "sentenceIndex": 2}, {"text": "opposes", "indexInDocument": 17, "posTag": "VBZ", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 18, "posTag": "DT", "sentenceIndex": 2}, {"text": "dismissal", "indexInDocument": 19, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 20, "posTag": "IN", "sentenceIndex": 2}, {"text": "MPON", "indexInDocument": 21, "posTag": "NNP", "sentenceIndex": 2}, {"text": "or", "indexInDocument": 22, "posTag": "CC", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 23, "posTag": "DT", "sentenceIndex": 2}, {"text": "MPOO", "indexInDocument": 24, "posTag": "NNP", "sentenceIndex": 2}, {"text": "confirmes", "indexInDocument": 25, "posTag": "VBZ", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 26, "posTag": "DT", "sentenceIndex": 2}, {"text": "dismissal", "indexInDocument": 27, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 28, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 29, "posTag": "DT", "sentenceIndex": 2}, {"text": "MPON", "indexInDocument": 30, "posTag": "NNP", "sentenceIndex": 2}, {"text": ".", "indexInDocument": 31, "posTag": ".", "sentenceIndex": 2}], "mentions": [{"type": "activity", "tokenDocumentIndices": [2]}, {"type": "activity", "tokenDocumentIndices": [11]}, {"type": "activity", "tokenDocumentIndices": [17]}, {"type": "activity", "tokenDocumentIndices": [25]}, {"type": "actor", "tokenDocumentIndices": [0, 1]}, {"type": "actor", "tokenDocumentIndices": [9, 10]}, {"type": "actor", "tokenDocumentIndices": [15, 16]}, {"type": "actor", "tokenDocumentIndices": [23, 24]}, {"type": "activity data", "tokenDocumentIndices": [3, 4]}, {"type": "activity data", "tokenDocumentIndices": [12, 13]}, {"type": "activity data", "tokenDocumentIndices": [18, 19, 20, 21]}, {"type": "activity data", "tokenDocumentIndices": [26, 27, 28, 29, 30]}, {"type": "xor gateway", "tokenDocumentIndices": [22]}, {"type": "condition specification", "tokenDocumentIndices": [23, 24, 25, 26, 27, 28, 29, 30]}], "entities": [{"mentionIndices": []}, {"mentionIndices": [5, 6, 7]}, {"mentionIndices": [8, 9, 10, 11]}, {"mentionIndices": [0]}, {"mentionIndices": [1]}, {"mentionIndices": [2]}, {"mentionIndices": [3]}, {"mentionIndices": [4]}, {"mentionIndices": [12]}, {"mentionIndices": [13]}], "relations": [{"headMentionIndex": 0, "tailMentionIndex": 1, "type": "flow"}, {"headMentionIndex": 1, "tailMentionIndex": 2, "type": "flow"}, {"headMentionIndex": 2, "tailMentionIndex": 12, "type": "flow"}, {"headMentionIndex": 12, "tailMentionIndex": 13, "type": "flow"}, {"headMentionIndex": 13, "tailMentionIndex": 3, "type": "flow"}, {"headMentionIndex": 0, "tailMentionIndex": 4, "type": "actor performer"}, {"headMentionIndex": 0, "tailMentionIndex": 8, "type": "uses"}, {"headMentionIndex": 0, "tailMentionIndex": 5, "type": "actor recipient"}, {"headMentionIndex": 1, "tailMentionIndex": 5, "type": "actor performer"}, {"headMentionIndex": 1, "tailMentionIndex": 9, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 6, "type": "actor performer"}, {"headMentionIndex": 2, "tailMentionIndex": 10, "type": "uses"}, {"headMentionIndex": 3, "tailMentionIndex": 7, "type": "actor performer"}, {"headMentionIndex": 3, "tailMentionIndex": 11, "type": "uses"}]}