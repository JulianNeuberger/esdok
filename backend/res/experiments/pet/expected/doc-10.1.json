{"nodes": [{"id": "0", "name": "The MPON", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, {"id": "2", "name": "the dismissal", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, {"id": "5", "name": "opposes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, {"id": "7", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}], "edges": [{"id": "0", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "1", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "2", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "0", "name": "The MPON", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "3", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "4", "source": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "5", "source": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "6", "source": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "7", "source": {"id": "5", "name": "opposes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "8", "source": {"id": "5", "name": "opposes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "9", "source": {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "opposes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "10", "source": {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "11", "source": {"id": "7", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "12", "source": {"id": "7", "name": "confirmes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MPOO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.1", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}]}